markasread/options.js

```diff
@@ -0,0 +1,87 @@
+var tcDefaults = {
```

Comment:
I'd like to see placeholders on the options page instead of defaults. 

For link color, I'm thinking of something like:
Link Color (default = blue) : [red, yellow, magenta...]

For the link list, I was thinking of having a list of placeholder values so that the user can input their own. We can use the ones that you already thought of.

Reply:
<img width="508" alt="Screenshot 2020-04-20 at 09 43 10" src="https://user-images.githubusercontent.com/973854/79726638-67e1d400-82eb-11ea-9c18-c94440fa545a.png">

<img width="304" alt="Screenshot 2020-04-20 at 09 43 17" src="https://user-images.githubusercontent.com/973854/79726664-729c6900-82eb-11ea-8366-6532c0feb685.png">

It's my fault for not being specific enough. This is what I would like for it to look like.